### PR TITLE
107

### PR DESCRIPTION
**Pull request by Aylr**

### CODE REVIEW

**Comment by levithatcher** [comment]
@Aylr Sorry for delay--is this ready for review?

**Comment by Aylr** [comment]
It is ready for review. In my haste to finish the conda stuff, the conda readme changes ended up in this branch. Since they are both ready for review, I'm not going to move them around.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # healthcareai
 
 [![Appveyor build status](https://ci.appveyor.com/api/projects/status/17ap55llddwe16wy/branch/master?svg=true)](https://ci.appveyor.com/project/CatalystAdmin/healthcareai-py/branch/master)
```
Comment (levithatcher) [inline README.md]: Great work. Love this! But do we need two badges?

File: docs/getting_started.md [hunk 2]
```diff
@@ -19,18 +19,17 @@
 - Run `conda remove scipy`
```
Comment (levithatcher) [inline docs/getting_started.md]: Thanks for remembering to do both--we need to make that standard practice for everyone. We need some sort of team best practices doc.

**Pull request merged**